phac-nml/irida-next
# Workflow submission file selection

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Currently, when a file is selected in the pipeline parameters dialog, if it is part of a pair, the second file is not selected.  This enhancement will automatically select the paired file in the second select dropdown.  If a single-ended file is selected, the second dropdown is cleared.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/11295750/b2e1f181-c334-49a6-bec7-e2b2c27ea6b1)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Go to a project with samples (http://localhost:3000/yersinia/yersinia-pseudotuberculosis/outbreak-2023/-/samples)
2. Create a new sample and upload a paired-end set of files to it
3. Go back to project samples and select the new sample 
4. Select a pipeline and see how the file selection is updated.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
